system/termcurses/tcurses_vt100.c

```diff
@@ -782,7 +782,7 @@ static int tcurses_vt100_move(FAR struct termcurses_s *dev, int type,
   FAR struct tcurses_vt100_s *priv;
   int   ret = -ENOSYS;
   int   fd;
-  char  str[16];
+  char  str[32];
```

Comment:
should we change `sprintf` to `snprintf`?

Reply:
Done.